.github/docs/VISUAL_ARCHITECTURE_GUIDE.md

```
@@ -0,0 +1,452 @@
+```
+TIME ???????????????????????????????????????????????????????????????
+
+0ms   App created
+      ?? MainPage = EmptyView
+      ?? InitializeApplicationAsync() started
+       ?
+        ?? Background: Init work...
+
+~10ms EmptyView.Loaded event fires
+      ?? Send ApplicationUiReadyMessage
+    ?? Navigate? (Window ready? Maybe... 50%?)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)